src/conf_mode/service_mdns-repeater.py

```diff
@@ -16,17 +16,20 @@
 
 import os
 
+from json import loads
 from sys import exit
 from netifaces import ifaddresses, interfaces, AF_INET
 
 from vyos.config import Config
+from vyos.ifconfig.vrrp import VRRP
 from vyos.template import render
 from vyos.util import call
 from vyos import ConfigError
 from vyos import airbag
 airbag.enable()
 
 config_file = r'/etc/default/mdns-repeater'
+vrrp_running_file = '/run/mdns_vrrp_active'
```

Comment:
Why not simply check if keepalived is running?

`vyos.util.process_named_running('keepalived')`

Reply:
That file is only created when `disable-on-vrrp` is set and there are VRRP groups. Only if it exists will the handler call the mdns repeater conf script on state changes.